### PR TITLE
VAN-1975 Added Azure Pipeline Modules and Helm Charts

### PR DESCRIPTION
**Pull request by shwetanshu-cldcvr**
Also, fixed dispatcher service Helm

### CODE REVIEW

File: helm/azure/cert-fetch/templates/issuer.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{{- if .Values.managedCertificate }}
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: letsencrypt
+  namespace: istio-system
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
```
Comment (dshelley66) [inline helm/azure/cert-fetch/templates/issuer.yaml:9]: what is acme?

**Comment by dshelley66** [comment]
@shwetanshu-cldcvr I thought this fix was going to correct the istio config issues that we encountered with Dispatcher earlier today? If so, I don't see any chances in here for the GCP templates?

File: helm/azure/cert-fetch/templates/issuer.yaml (hunk 1 above)
Comment (shwetanshu-cldcvr) [reply helm/azure/cert-fetch/templates/issuer.yaml:9]: This is the LetEncrypt certificate provider. It automatically issues a certificate free of cost.

File: helm/azure/istio-aks/config.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
+{
```
[the remark this answers is not in the document]
Comment (shwetanshu-cldcvr) [reply helm/azure/istio-aks/config.json:1]: This isn't needed, I'll remove this file.

File: pipeline-modules/deployment-service/gcp/istio-gke-regional.yaml [hunk 3]
```diff
@@ -359,7 +359,12 @@ template: |
       applicationName: {{applicationName}} #auto
       environment:  {{environment}} #auto
       project: {{project}} #auto
-      istioInjection: {{ istioInjection | lower }}
+      {% if istioInjection %}
+      istioInjection: enabled
+      {% else %}
+      istioInjection: disabled
+      {% endif %}
+      {% if deployment %}
```
Comment (dshelley66) [inline pipeline-modules/deployment-service/gcp/istio-gke-regional.yaml:367]: seems like this is an extra "if"?

**Review by dshelley66 — APPROVED** [review]

**Comment by dshelley66** [comment]
@shwetanshu-cldcvr you can merge this - the issues on the guardrails scan were all false positive...

**Pull request merged**